sklearn/neighbors/_binary_tree.pxi

```diff
@@ -166,6 +166,7 @@ from ..utils._typedefs import DTYPE, ITYPE
 from ..utils._heap cimport heap_push
 from ..utils._sorting cimport simultaneous_sort as _simultaneous_sort
 
+# TODO: use cnp.PyArray_ENABLEFLAGS when cython >= 3.0 is used.
```

Comment:
To ease searches when performing the migration to Cython>=3.0
```suggestion
# TODO: use cnp.PyArray_ENABLEFLAGS when Cython>=3.0 is used.
```

Reply:
I propose only treating this, and have another PR to treats the removal of duplicate `cnp.darray`s and `_arr`.

What do you think?